src/sigma.core.js

```diff
@@ -2,6 +2,16 @@
   'use strict';
 
   var __instances = {};
+  
+  // Deal with resize:
+  window.addEventListener('resize', function() {
+    for (var key in __instances) {
+      if (__instances.hasOwnProperty(key)) {
+        var graph = __instances[key];
```

Comment:
Can you rename `graph` to `instance` please? This is a reference to a sigma instance and not a graph instance.